src/mudlet.cpp

```diff
@@ -1179,6 +1172,20 @@ void mudlet::disableToolbarButtons()
     mpMainToolBar->actions()[12]->setEnabled(false);
     mpMainToolBar->actions()[13]->setEnabled(false);
     mpMainToolBar->actions()[14]->setEnabled(false);
+
+    mpActionReplay->setEnabled(false);
+    mpActionReplay->setToolTip(QStringLiteral("<html><head/><body>%1</body></html>")
+                               .arg(tr("<p>Load a Mudlet replay.</p>"
+                                       "<p><i>Disabled until a profile is loaded!</i></p>")));
```

Comment:
Don't really need the second message to show all the time - if you'd like, show it when it is relevant, when no profile is loaded (but without the exclamation mark, it's not an exciting thing)

Reply:
It does __not__ show all the time - only when it is disabled.

Reply:
OK. No exclamation mark then please, there is nothing exciting about a disabled button.